### ISSUE TITLE
Reduce verbosity of screen reader information region

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#9903]
The screen reader region should be shorter, or we should hide it behind an interaction (popup, menu etc), so that screen reader users can get to the chart quickly.

**Comment by oysteinmoseng** [comment]
- Remove redundant headings (title spoken multiple times - <title> tag, <h3>, and aria label.
- Remove screen reader navigation hint
